mautic/mautic
# Fix lookup properties

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ✅ 
| New feature/enhancement? (use the a.x branch)      | 
| Deprecations?                          | 
| BC breaks? (use the c.x branch)        | 
| Automated tests included?              | 
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 



#### Description:

This PR fix properties list for lookup field type. On edit field you can see not flipped values

![image](https://user-images.githubusercontent.com/462477/176097613-6dc54632-2094-4351-b8f6-2e97b9dace26.png)




#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Try edit Title field from Custom Fields
3. Before PR you see number in properties list, after PR you should see regular strings




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11296"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
